### DISCUSSION TITLE
Filament 2 loading issue when creating or editing a record

### DISCUSSION THREAD

**Discussion by ouziine** [filamentphp/filament#7622] [Help] [answered]
the creation process in the modal takes forever to get saved in the hosting (production)
https://drive.google.com/file/d/1mKbfSS7HmFouknG4ZukHqRdK51DybpfF/view?usp=drive_link

https://drive.google.com/file/d/1E3gGdFo_lKyKrgpuGh_IiitcfhdxBXkU/view?usp=drive_link

the video in the drive 
please help it's urgent

**Comment by danharrin** [comment]
There is no way for us to help with so little detail. Put more effort in to your questions.

**Comment by ouziine** [comment, marked as answer]
i was listing all db table rows in the listing table 
so the database request gets slower
